tripy/tests/integration/test_functional.py

```diff
@@ -78,59 +78,6 @@ def test_multi_output_trace(self):
             and (cp.from_dlpack(out[1]) == cp.array([4.0, 4.0], dtype=np.float32)).all()
         )
 
-    def _test_framework_interoperability(self, data, device):
-        a = tp.Tensor(data, device=device)
-        if device.kind == "gpu":
-            b = tp.Tensor(torch.tensor(data).to("cuda"), device=device)
-        else:
-            b = tp.Tensor(torch.tensor(data), device=device)
-
-        out = a + b
-        assert (cp.from_dlpack(out).get() == np.array([2.0, 2.0], dtype=np.float32)).all()
-
-    def test_cpu_and_gpu_framework_interoperability(self):
```

Comment:
Why are we removing this? Is it because we have coverage elsewhere?

Reply:
Yes, in `tripy/tests/frontend/test_tensor.py`, I added tests for `dlpack` interface.